### DISCUSSION TITLE
No networking send and receive data

### DISCUSSION THREAD

**Discussion by cliffordsnow** [cockpit-project/cockpit#19819] [General]
I'm running two different instances of Cockpit, one on Ubuntu 22.04 the other on Fedora F38. Neither show any sending and receiving data either in the graph or the interfaces table. 

Otherwise both are working great. Since there are two different linux operating systems, I suspect I missed a configuration. Any suggestions to how to configure them to show the data activity?

**Comment by cliffordsnow** [comment]
I was able to get the Fedora F38 to work. pcp needed to be installed and pmcd.service started. Interesting because Fedora Server came with Cockpit preinstalled.

Ubuntu still isn't showing any network activity after installing pcp and making sure pmcd.service was running.

**Comment by cliffordsnow** [comment]
Ubuntu - fixed. Missed installing cockpit-pcp.

Hope this helps someone in the future

**Comment by cliffordsnow** [comment]
closing issue

**Comment by sachron3** [comment]
thanks for your advice
